tap_pipedrive/tap.py

```diff
@@ -248,7 +248,7 @@ def execute_stream_request(self, stream):
         backoff.expo,
         (ConnectionError, RequestException, HTTPError),
         max_tries=5,
-        giveup=lambda e: e.response is not None and 400 <= e.response.status_code,
+        giveup=lambda e: e.response is not None and 400 >= e.response.status_code,
```

Comment:
This is counter intuitive

Reply:
So does it say:
"I will giveup the retries, if the reponse status code is higher or equal than 400"?

Reply:
its actually saying 
.......e.response is not None ..................and 400 >= e.response.status_code
give up if:  has response (is http error) and statuscode is less than or equal to 400

Reply:
and yes, not good writing, should i refactor?

Reply:
Hahahahaha oh my god, I totally read it wrong. I read left to right status_code >= 400, and I was like wtf, because I'm used to that. Sorry for confusing you Johannes!

Reply:
You don't have to refactor, this is correct :)